packages/expo-cli/src/credentials/utils/provisioningProfile.ts

```diff
@@ -22,4 +15,19 @@ function readAppleTeam(dataBase64: string): AppleTeam {
   return { teamId, teamName };
 }
 
-export default { readAppleTeam };
```

Comment:
Is there any any specific reason to drop default ?

Reply:
Yes. If I wanted to only import `readAppleTeam` I would have to import `readProfileName` too. This is why using `export default` in such cases doesn't make sense.